doc/help/commands.asciidoc

```diff
@@ -735,6 +735,7 @@ Save a session.
 * +*-c*+, +*--current*+: Save the current session instead of the default.
 * +*-q*+, +*--quiet*+: Don't show confirmation message.
 * +*-f*+, +*--force*+: Force saving internal sessions (starting with an underline).
+* +*-o*+, +*--only-active-window*+: Saves only tabs of the currently active window.
```

Comment:
Docs are generated from the docstrings in the code by running `scripts/dev/src2asciidoc.py`. Please edit the docstring instead and then run that script (or revert those changes and I'll run it when merging).